charts/influxdb-enterprise/README.md

```diff
@@ -0,0 +1,82 @@
+# InfluxDB Enterprise
+
+## QuickStart
+
+```bash
+helm repo add influxdata https://helm.influxdata.com/
+helm upgrade --install influxdb influxdata/influxdb-enterprise --namespace monitoring
+```
+
+> **Tip**: `helm upgrade --install [RELEASE] [CHART] [FLAGS]` can be shortened : `helm upgrade -i [RELEASE] [CHART] [FLAGS]`
+
+## Introduction
+
+This chart bootstraps an InfluxDB Enterprise cluster, with a StatefulSet for both the Meta and Data nodes.
+
+## Prerequisites
+
+- Kubernetes 1.4+
+- PV provisioner support in the underlying infrastructure (optional)
+
+### Secrets
+
+This chart REQUIRES some mandatory secrets in-order to function.
+
+#### License
+
+InfluxDB Enterprise requires a license. To provide the license, you can either store it in a secret or provide a string within your `values.yaml`. We recommend using a secret.
+
+```yaml
+license:
+  # You can put your license key here for testing this chart out,
+  # but we STRONGLY recommend using a license file stored in a secret
+  # when you ship to production.
+  # key: "your license key"
+  secret:
+    name: license
+    key: json
+```
+
+#### Shared Secret
+
+The meta cluster requires a shared internal secret to secure communication. This must be provided by specifying a secret name in the `values.yaml` file.
+
+The secret MUST container a key called `secret` that is a randomly generated string.
```

Comment:
`contain`, auto-correct making you type container. Does this mean we should have a layout like this?

```
meta:
  sharedSecret:
    secret: aa8c346b84418626b91e1f66c14230f7
```

Reply:
Reading further I think you need doc update.